homeassistant/config_entries.py

```diff
@@ -192,6 +192,15 @@ def recoverable(self) -> bool:
     "config_entry_changed"
 )
 
+
+@lru_cache
```

Comment:
```suggestion
@cache
```

We can use the fast path as there is no need to evict